techscreenr-angular/client/app/app.component.html

```diff
@@ -1,5 +1,4 @@
 <div class="container">
-
   <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
@amartinelli12  Are we still using bootstrap on this project? I thought we were going Angular Material and flex boxes all the way?